components/Media/src/Dependencies.props

```diff
@@ -11,12 +11,15 @@
 <Project>
   <!-- WinUI 2 / UWP -->
   <ItemGroup Condition="'$(IsUwp)' == 'true'">
-    <PackageReference Include="Win2D.uwp" Version="1.28.0" />
-    <PackageReference Include="System.Threading.Tasks.Extensions" Version="4.6.0" />
+    <PackageReference Include="Win2D.uwp" Version="1.28.1" />
+
+    <!-- Only include this polyfill package when not targeting .NET 8 or above -->
+    <PackageReference Condition="!$([MSBuild]::IsTargetFrameworkCompatible('$(TargetFramework)', 'net8.0'))"
```

Comment:
You'll want to double check that this doesn't break when you build the UWP gallery head. It's not SDK style, which causes problems with Conditions when running in VS. We've had to work around it [like this](https://github.com/CommunityToolkit/Tooling-Windows-Submodule/blob/main/MultiTarget/PackageReferences/Uwp/Microsoft.UI.Xaml.props) before.

Reply:
I think it should be fine, it'll just always add this package. Which won't break anything (at worst it's just redundant).